test/testUnregisterMachine.js

```diff
@@ -67,26 +92,29 @@ describe('mercurius unregisterMachine', function() {
       token: token,
       machineId: 'machine_2',
     })
-    .expect(200, done);
+    .expect(404, done);
```

Comment:
I don't like it sends 404. 
It happens only if unregistering itself from the button where it is not required to recieive a value.
I consider also a condition in sendMachines to not check for 404 and instead send an empty array...
That thing is at least debatable for me